### DISCUSSION TITLE
Discussions are open!

### DISCUSSION THREAD

**Discussion by erosDiffusion** [erosDiffusion/ComfyUI-enricos-nodes#67] [Announcements]
Ffeel free to request features, post rants or appreciation, link your resources, showcase your work.
Saz what you like, what you don't and how you envision the plugin!

**Comment by Poisy00** [comment]
Hi I would Like to use Fluxmania model but Im facing issues Like : "VAEEncode

ERROR: VAE is invalid: None



If the VAE is from a checkpoint loader node your checkpoint does not contain a valid VAE.", "CLIPTextEncode

ERROR: clip input is invalid: None



If the clip is from a checkpoint loader node your checkpoint does not contain a valid clip or text encoder model."
